### ISSUE TITLE
add new prop showChat set defualt chatOpened state

### ISSUE THREAD

**Issue by gwmaster** [Wolox/react-chat-widget#20]
cool component
i need that when component is loaded the default is show chat
thanks 👍

**Comment by mcallegari10** [comment]
Hi! Thanks for the advise on the Frame component, never heard of it but I'll definitely check it out. The first feature should be accomplished with the `toggleWidget()` function when the parent component is rendered. And the second one is actually a good one. I had something similar in mind for this kind of feature, I'll try to add it in the next couple of days :+1:

**Comment by charlesj** [comment]
Yes, I am running into this as well (similar to #27).  I just referenced your component on one of my pages, and now all the styles are everywhere on every page.  The problem seems to be here: 

https://github.com/Wolox/react-chat-widget/blob/master/src/components/Widget/components/Conversation/components/Header/style.scss#L4

Which uses very generic names like 'header' or 'title'.  Those are also used by other frameworks as well (like Semantic UI, for example) so when those classes load, they are applied to many things.

Would you accept a PR that changes them to have more component specific names?

**Comment by gwmaster** [comment]
try use this solution 
https://medium.com/@bluepnume/creating-a-cross-domain-react-component-with-xcomponent-fbcccc4778fd
